### PR TITLE
[DeltaGreenRPG] Updated character sheet and minor fix

### PR DESCRIPTION
**Pull request by uriele**
## Changes / Comments
- Add versioning for future updates
- Add low wp reminder in roll template when WP<=2
- Fixed autocomputation of ChaX5
- At character creation, it is possible to setup the current san and BP by changing the value in Pow (after the value has been changed once, subsequent changes will not modify the current san or the bp value as by the rules)
- Small aesthetic change in the rolltemplates.
- Updated the JSON instructions for new users.

## Roll20 Requests

Comments are very helpful for reviewing the code changes. Please answer the relevant questions below in your comment.

- [x] Does the pull request title have the sheet name(s)? Include each sheet name.
- [x] Is this a bug fix?
- [x] Does this add functional enhancements (new features or extending existing features) ?
- [x] Does this add or change functional aesthetics (such as layout or color scheme) ? 
- [x] If changing or removing attributes, what steps have you taken, if any, to preserve player data ?
- [ ] If this is a new sheet, did you follow [Building Character Sheets standards](https://wiki.roll20.net/Building_Character_Sheets#Roll20_Character_Sheets_Repository) ?

If you do not know English. Please leave a comment in your native language.

### CODE REVIEW

**Comment by Anduh** [comment]
So you simplified the roll templates, and there is a good reason several thousand lines of them was straight up deleted? Just asking, as the amount of code removed seem fairly sizable.

**Comment by uriele** [comment]
Oh no it's a fair question: 
So these sheet was build on top of my original Delta Green one (that has been discontinued). If you check the code I simply removed the rolltemplate called sheet-rolltemplate-dg-attack that contained both lethality and damage attack and was using complex rollGreater to check if the attack was lethal or normal.

These roll template wasn't used in the new version and has been replaced by 4 different ones:
1) sheet-rolltemplate-dgattackfire that covers all attack rolls except shotguns (that is an exception), lethality, and melee/hand-to-hand attacks
2) sheet-rolltemplate-dgattacklethal that covers all lethality attack
3) sheet-rolltemplate-dgattackhand that covers all melee/hand-to-hand attacks
4) sheet-rolltemplate-dgattackshotgun that covers the shotgun attacks

TL;DR: It was vestigial code that wasn't used nor mantained, so I removed it :)

**Comment by uriele** [comment]
Ahahah! Thanks, I also double checked I didn't cancel anything important and tested the sheet multiple times :D
